WebDriverAgentLib/Categories/XCUIElement+FBFind.m

```diff
@@ -13,21 +13,38 @@
 #import "FBElementTypeTransformer.h"
 #import "XCElementSnapshot.h"
 #import "XCElementSnapshot+FBHelpers.h"
+#import "XCUIElement+FBUtilities.h"
 #import "XCUIElement+FBWebDriverAttributes.h"
 
 @implementation XCUIElement (FBFind)
 
++ (NSArray<XCUIElement *> *)lookupDescendants:(XCUIElementQuery *)query shouldReturnAfterFirstMatch:(BOOL)shouldReturnAfterFirstMatch
```

Comment:
This is category so you should add `fb_` prefix. I would also consider different name, maybe
```
+ (NSArray<XCUIElement *> *)fb_extractMatchingElemetnsFromQuery:(XCUIElementQuery *)query shouldReturnOnlyFirstMatch:(BOOL)shouldReturnAfterFirstMatch
```
?